Maestro/subscriptions.json

```diff
@@ -6,6 +6,13 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 1084
     },
+    // A build definition capable of running any .ps1 script in the CoreFX repo dev/api branch
+    "corefx-general-dev-api": {
+      "vsoInstance": "devdiv.visualstudio.com",
+      "vsoProject": "DevDiv",
+      "vsoSourceBranch": "dev/api",
```

Comment:
No, sorry on the lack of doc here.

The `vsoSourceBranch` goes into the under the "subscriptions" section.  You don't need to create a new "action" here.

See https://github.com/dotnet/versions/blob/master/Maestro/test/test-subscriptions.json for an example.